src/reducers/index.js

```diff
@@ -46,6 +46,9 @@ export const getKonnectorTriggersCount = (state, konnector) =>
 export const getKonnectorsInMaintenance = () =>
   fromKonnectors.fetchKonnectorsInMaintenance()
 
+export const isInMaintenance = slug =>
+  !!fromKonnectors.fetchKonnectorsInMaintenance()[slug]
```

Comment:
It's better to place `isInMaintenance` in '../ducks/konnectors' IMHO.

Also, we should not do this `fetchKonnectorsInMaintenance()[slug]`